docs/conf.py

```diff
@@ -43,6 +43,7 @@
     "sphinx.ext.githubpages",
     "sphinx.ext.napoleon",
     "sphinxcontrib.bibtex",
+    "sphinx_copybutton",
```

Comment:
This is a new dependency which lets a user copy a codeblock in the documentation.